projects/demo/src/modules/components/dialog/examples/11/index.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, inject, ViewEncapsulation} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {TuiMetaColorService} from '@taiga-ui/cdk';
+import {TuiButton, TuiDialogService} from '@taiga-ui/core';
+
+@Component({
+    standalone: true,
+    imports: [TuiButton],
+    templateUrl: './index.html',
+    encapsulation: ViewEncapsulation.None,
+    changeDetection,
+})
+export default class Example {
+    private readonly dialogs = inject(TuiDialogService);
+    private readonly themeColor = inject(TuiMetaColorService);
+
+    protected showDialog(): void {
+        const [prev] = this.themeColor.setColor('#ffdd2d');
```

Comment:
I wonder if we should have a token for initial value and bundle it into dialogs 🤔